engine-comparison/dispatcher.sh

```diff
@@ -5,61 +5,96 @@
 . $(dirname $0)/../common.sh
 . ${SCRIPT_DIR}/common-harness.sh
 
+
 build_engine() {
-  # [[ ! -e ~/$FENGINE_CONFIGS_DIR/$1 ]] && echo "cant do" && exit 1
-  # All of this code will probably be in a separate script
 
   FENGINE_CONFIG=$1
-
+  [[ ! -e $FENGINE_CONFIG ]] && echo "File DNE" && exit 1
   echo "Building $FENGINE_CONFIG"
   FENGINE_NAME=$(basename $FENGINE_CONFIG)
-  ENGINE_DIR=$WORK/fuzz-engines/${FENGINE_NAME}
-  rm -rf $ENGINE_DIR
-  mkdir $ENGINE_DIR
-
-  # Fuzzing activities go here
+  FENGINE_DIR=$WORK/fengine-builds/${FENGINE_NAME}
+  rm -rf $FENGINE_DIR
+  mkdir $FENGINE_DIR
+
+  . $FENGINE_CONFIG
+  # Build either engine
+
+  if [[ $FUZZING_ENGINE == "libfuzzer" ]]; then
+    echo "Making a version of libfuzzer"
```

Comment:
Ok, so this was a good catch. There is an asymmetry because `build_fuzzer` builds libfuzzer, but it doesn't actually _build_ for AFL, it just moves linked libraries.

The AFL method may make more sense in this context, as the current calls to `build_libfuzzer` will require rebuilding an identical version of libfuzzer `B` times if we have `B` benchmarks. As it stands right now, however, we'll just checkout libfuzzer once and be done; further changes/improvements would require changing ~two dozen `build.sh` files